tests/cli/conftest.py

```diff
@@ -32,3 +32,15 @@ def do_run(*args):
         return testdir.run(*args)
 
     return do_run
+
+
+@pytest.fixture
+def run_in_default_project_with_info(testdir: Testdir) -> Callable[..., RunResult]:
+    os.environ["LOG_LEVEL"] = "INFO"
```

Comment:
shouldn't `info` be the default log level?

Reply:
It is, but if another test sets the `os.environ["LOG_LEVEL"] = "ERROR"` it stays this way for the next test and thus breaks it.

Reply:
That's why I didn't realize that "INFO" was the default. I now changed the `run_in_default_project` routine to reset it after running.